asttokens/mark_tokens.py

```diff
@@ -328,11 +329,14 @@ def visit_const(self, node, first_token, last_token):
   visit_constant = visit_const
 
   def visit_keyword(self, node, first_token, last_token):
-    if node.arg is not None:
-      equals = self._code.find_token(first_token, token.OP, '=', reverse=True)
-      name = self._code.prev_token(equals)
-      util.expect_token(name, token.NAME, node.arg)
-      first_token = name
+    # ast.keyword changed in 3.9 https://bugs.python.org/issue40141
+    # Astroid was not affected
+    if sys.version_info < (3, 9) or not isinstance(node, ast.AST):
```

Comment:
Thank you! I looked a bit closer, and would like to propose this condition instead, which seems more meaningful. I checked that it works:

```
+    # Until python 3.9, ast keyword nodes didn't have line info. Astroid has lineno of None.
+    if node.arg is not None and getattr(node, 'lineno', None) is None:
```

(if it holds, apply old logic, else it's no longer needed)

Reply:
Great! I'm surprised the attribute was actually missing before.